crates/bevy_text/src/font_atlas.rs

```diff
@@ -8,40 +7,9 @@ use bevy_render::{
 use bevy_sprite::{DynamicTextureAtlasBuilder, TextureAtlas};
 use bevy_utils::HashMap;
 
-#[cfg(feature = "subpixel_glyph_atlas")]
-#[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
-pub struct SubpixelOffset {
-    x: u16,
-    y: u16,
-}
-
-#[cfg(feature = "subpixel_glyph_atlas")]
-impl From<Point> for SubpixelOffset {
-    fn from(p: Point) -> Self {
-        fn f(v: f32) -> u16 {
-            ((v % 1.) * (u16::MAX as f32)) as u16
-        }
-        Self {
-            x: f(p.x),
-            y: f(p.y),
-        }
-    }
-}
-
-#[cfg(not(feature = "subpixel_glyph_atlas"))]
-#[derive(Copy, Clone, Eq, PartialEq, Hash, Debug)]
-pub struct SubpixelOffset;
-
-#[cfg(not(feature = "subpixel_glyph_atlas"))]
-impl From<Point> for SubpixelOffset {
-    fn from(_: Point) -> Self {
-        Self
-    }
-}
-
 pub struct FontAtlas {
     pub dynamic_texture_atlas_builder: DynamicTextureAtlasBuilder,
-    pub glyph_to_atlas_index: HashMap<(GlyphId, SubpixelOffset), usize>,
+    pub glyph_to_atlas_index: HashMap<cosmic_text::CacheKey, (usize, i32, i32, u32, u32)>,
```

Comment:
I would really like a newtype here rather than the complex tuple type. Those are really easy to screw up.